script.js

```diff
@@ -44,32 +41,108 @@ class ToDoList {
     }
 }
 
+// Currently we set up one default list, but it would be relatively straightforward
+// to add functionality to allow the user to create multiple lists.
 const toDoList = new ToDoList(true);
 
-// Joe: update display/DOM
 
-// Joe: update item in toDoList (to set it as "done")
+function updateDisplay(list) {
+
+    const outputElement = document.querySelector('section');
+    const items = list.getAllItems();
+    outputElement.innerHTML = '';
+
+    for (const item of items) {
+
+        const taskContainerEl = document.createElement('div');
 
-// (Stretch goal: hide completed items)
+        const toggleDoneInputEl = document.createElement('input');
+        toggleDoneInputEl.setAttribute('type', 'checkbox');
 
-/*
+        const taskTextContainerEl = document.createElement('span'); // change to div?
 
-    Add, update and delete functions *only* change the array.
-    (They only change the data that we're storing.)
+        // If the item's done flag is set to true:
+        if (item['done'] === true) {
```

Comment:
Why did you choose to use bracket notation here? Maybe using dot notation to access `done` would be better since we already know which property to access.

Reply:
I just tend to use bracket notation when I remember to. (Because bracket notation can always be used, but dot notation can only be used in certain cases.)

Totally don't mind using dot notation instead, if you'd prefer? :)

Reply:
Well, it might be a good way to avoid bugs since the bracket notation is not strict, but for this project should be fine :)

Reply:
Do you have any references for the strictness/safety of dot vs bracket notation? I haven't heard about this before 👀 

(Either way, will change to dot for this :)

Reply:
Ah, I see. Thanks for explaining!